documentation/book/proc-upgrading-brokers-newer-kafka.adoc

```diff
@@ -55,7 +65,7 @@ kind: Kafka
 spec:
   # ...
   kafka:
-    version: 2.1.0 <1>
+    version: 2.1.1 <1>
     config:
       log.message.format.version: "2.0" <2>
```

Comment:
As with the downgrade procedure this one has hard coded versions in the examples, but parametrised versions in the text.

Reply:
I've updated the code blocks to use attributes for the Kafka versions.